cloud/google/gce.py

```diff
@@ -87,10 +87,18 @@
       - your GCE project ID
     required: false
     default: null
-  name:
+  base_name:
```

Comment:
@jbaublitz, thanks for this.  Spoke with @ryansb and we need to keep name.  I'd like to have base_name in-addition-to, rather than instead-of name.  Sound good?